ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -67,6 +135,26 @@ const ListingCard = (props: ListingCardProps) => {
             {t("t.seeDetails")}
           </LinkButton>
         )}
+        <>
+          {profile &&
+            (updatedFavorites?.includes(props.listing.id) ? (
+              <Button
```

Comment:
The button is kinda oval shaped, and to the left of the info column. Could you either match the mocks in this PR, or leave a TODO to fix the UI?